.node-version

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I have a preference for exact versions elsewhere but this is fine here, I assume, and package-lock should pin down a version, right?

Reply:
Very happy to be exact here -- package-lock won't pin down a version of node.  I'll update.

Reply:
Just to expand a bit more -- package-lock is only for locking down npm dependencies.

This file is used by nothing out of the box; I personally like to run `ln -s .node-version .nvmrc` in my projects since I use nvm, and then nvm will install whatever version is specified in this file.

Some Node projects commit `.nvmrc` directly but I personally think of that file as a local development dependency (not everybody uses nvm / there are other ways to manage local node versions)

`.node-version` is a clear enough file name!

Hope that helps explain better than "it won't do that" 😅

Reply:
If used by nothing, maybe it's superfluous. But I guess it makes sense to be specific. And yes, it makes sense that package-lock will only be used by npm.

Reply:
Well it *is* used by humans in the sense that it documents what version of node we are targeting for development so I do think it serves a purpose (otherwise someone setting up would need to ask us what version of node to use)